code/modules/multiz/turf.dm

```diff
@@ -77,8 +78,13 @@
 /turf/simulated/open/update_icon()
 	cut_overlays() // Edit - Overlays are being crashy when modified.
 	update_icon_edge()// Add - Get grass into open spaces and whatnot.
-	var/turf/below = GetBelow(src)
 	if(below)
+		// Skybox lives on its own plane, if we don't set it to see that, then open space tiles over true space tiles see white nothingness below
```

Comment:
This should work, but does make me consider the value of caching is_space() someday.